KarhooUISDK/Translations/es.lproj/Localizable.strings

```diff
@@ -8,6 +8,8 @@
 
 "ELECTRIC" = "Eléctrico";
 
+"EXEC" = "";
```

Comment:
All of the empty pre-existing keys used to have values. They still do on the develop branch, but they seem to be missing now in Phrase. Please re-add the values to all the keys that appear in this PR as being modified (except for the legal notice related ones) and re-update the strings files.

Reply:
I cannot find values for this keys neither in Phrase.com nor in develop. To be able download apty values for legal notice I checked "include empty strings" while downloading translations. That's reason why new empty keys appeared.

Reply:
`"EXEC"` And keys like this one has been removed from phrase.com. Full list of removed keys can be found here: 
https://karhoo.atlassian.net/browse/MOB-3703

Reply:
translations are updated, keys are removed